assets/queries/dockerCompose/restart_policy_on_failure_not_set_to_5/metadata.json

```
@@ -0,0 +1,10 @@
+{
+    "id": "2fc99041-ddad-49d5-853f-e35e70a48391",
+    "queryName": "Restart Policy On Failure Not Set To 5",
```

```suggestion
    "queryName": "Unrecommended Restart Policy On Failure",
```

Please, also update the query folder name

---

I disagree, I think we should keep the current query name.